### DISCUSSION TITLE
When Do the Betas Get Uploaded to the Google Play Store?

### DISCUSSION THREAD

**Discussion by SabinVI** [jellyfin/jellyfin-androidtv#4219] [General]
Hello. I'm on the Beta track on my Sony Android TV and also for my Nvidia Shield. I've been waiting for weeks to test v18 beta but it still hasn't been uploaded to the Google Play store. I saw a note on Beta 1 saying that it would be there in about a week after release here, but I guess that never happened.

I know I can sideload it, but I really enjoy the convenience of the auto updates. Once you sideload you are stuck on that version if I remember correctly. Until you sideload again or reload from Play store. I've tried that before and I'm just trying to avoid dealing with that again.

You guys are doing great work and v18 looks amazing. I hope you'll get it on the Play store soon! Thanks for everything.

**Comment by SabinVI** [comment]
Of course… I literally got the beta on my TV tonight less than hour after posting this. I don’t know if you guys did anything or that was just a huge coincidence. Either way, thank you!

**Comment by nielsvanvelzen** [comment]
Information about the beta releases is documented in [this blog post](https://jellyfin.org/posts/android-betas/). In addition we use a "slow rollout" which means not everyone will get the release directly.
The first beta for 0.18 was skipped entirely on Google Play, the others all had a slow rollout. Whenever we hit some big issue the rollout will either be stopped at a specific percentage or completely halted. This is what happened for the second beta. The third beta is currently rolling out to nearly everyone in the beta channel.
